shortcodes/epfl-labs-search/controller.php

```diff
@@ -5,10 +5,17 @@
 /**
  * render the shortcode, mainly a form and his table
  */
-function renderLabsSearch($sites, $faculty, $institute, $field) {
+function renderLabsSearch($sitesList, $faculty, $institute, $field) {
   wp_enqueue_script( 'lib-listjs', plugins_url('lib/list.min.js', dirname(__FILE__)), ['jquery'], 1.5, false);
   wp_enqueue_style( 'epfl-labs-search-css', plugins_url('epfl-labs-search.css', __FILE__),false,'1.1','all');
 
+  $titles = array_column($sitesList, 'title');
```

Comment:
```suggestion
  // because the sites array arrives sorted case sensitively, do a "natural order" to sort case-insensitively.
  $titles = array_column($sites, 'title');
```

Reply:
- un petit commentaire pour nos futurs nous.
- je retransforme les $sitesList en $sites